### DISCUSSION TITLE
usage with vite

### DISCUSSION THREAD

**Discussion by DanavsSite** [sushinpv/react-secure-storage#16] [General]

**Comment by gregdusoli** [comment]
No one has a tip on how to use this lib with Vite yet?

**Comment by ryanrdetzel** [comment]
You need to set this in your vite.conf

```
import { defineConfig } from 'vite'
// ...
export default defineConfig({
  // ...
  define: {
    'process.env': {}
  }
})
```

**Reply by UnbreakableKid** [reply]
how do I setup the env variables (prefix for example) after? like a normal VITE env var?

**Reply by sushinpv** [reply]
Hi @UnbreakableKid We have added the support for VITE_ env's in the latest version 1.2.1
